sycl/include/sycl/ext/oneapi/reduction.hpp

```diff
@@ -537,7 +537,7 @@ class reduction_impl_algo<
                ext::oneapi::accessor_property_list<>>;
 
   static constexpr bool has_atomic_add_float64 =
```

Comment:
I think the variable itself has to be renamed, based on its uses.

Reply:
Thanks @aelovikov-intel, good point!

I did it in the following commits.